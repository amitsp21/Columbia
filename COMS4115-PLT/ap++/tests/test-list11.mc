int main()
{
  list<int> a;
  int i;
  for (i=0; i<5; ++i) { 
    list_push(a, i); 
  }

  list_insert(a, 0, 99);
  printi(#a);
  for (i=0; i<#a; ++i) { 
    printi(a[i]);
  }
  
  list_insert(a, #a-1, 101);
  printi(#a);
  for (i=0; i<#a; ++i) { 
    printi(a[i]);
  }

  list_insert(a, #a, 102);
  printi(#a);
  for (i=0; i<#a; ++i) { 
    printi(a[i]);
  }

  list_insert(a, 2, 103);
  printi(#a);
  for (i=0; i<#a; ++i) { 
    printi(a[i]);
  }

  return 0;
}