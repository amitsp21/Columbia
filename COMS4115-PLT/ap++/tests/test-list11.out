6
99
0
1
2
3
4
7
99
0
1
2
3
101
4
8
99
0
1
2
3
101
4
102
9
99
0
103
1
2
3
101
4
102